### DISCUSSION TITLE
Pull request steps

### DISCUSSION THREAD

**Discussion by Leni-Yeo** [idaholab/moose#27863] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Good morning.
I have a general question about "Pull requests". 
I submitted one (#27816) for the first time recently.
I saw that there are checks that have failed (_46 errored and 1 failing checks_), so I tried to see the details next to them, but the results screen doesn't load/show anything even after a day. I don't know if I have something to do/fix anything on my end or if I just have to wait till the developers complete a process first.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Thanks for reaching out. I replied there with more information.
The output of the testing is already in the PR, but maybe you could not see it

Guillaume
